### DISCUSSION TITLE
using strongSwan libraries for iOS

### DISCUSSION THREAD

**Discussion by MewanCdot** [strongswan/strongswan#2500] [Q&A] [answered]
I want to use the strongswan libraries for building an iOS VPN app but i dont want to use the Network Extension framework becuase i am integrating liboqs with the strongswan libraries, and i am not well acquainted with the Network Extension framework. Is there any way to do this?

**Comment by tobiasbrunner** [comment, marked as answer]
This is a duplicate of #877, check my replies there.

**Reply by MewanCdot** [reply]
alright thanks
